kiota_abstractions/utils.py

```diff
@@ -0,0 +1,26 @@
+import importlib.util
+import sys
+
+
+def lazy_import(name):
+    """Lazily imports a python module given its absolute path
```

Comment:
We should probably indicate in the doc comment that this method is only meant for the generated code and that people should not take a dependency on it outside of that scenario.

Reply:
Added in https://github.com/microsoft/kiota-abstractions-python/pull/19/commits/aa4c29726a09d763c66181f694dd13c3bb7be701